server/form-pages/apply/risks-and-needs/risk-of-serious-harm/riskFactors.ts

```diff
@@ -35,12 +57,32 @@ export default class RiskFactors implements TaskListPage {
   errors() {
     const errors: TaskListErrors<this> = {}
 
+    if (!this.body.confirmation) {
+      errors.confirmation = errorLookups.oasysConfirmation.empty
+    }
+    if (!this.body.circumstancesLikelyToIncreaseRisk) {
+      errors.circumstancesLikelyToIncreaseRisk = 'Enter the circumstances that are likely to increase risk'
```

Comment:
is it worth adding these to the error lookups file to avoid repetition between here and the test file?